.github/workflows/Procedure_push_docker_images.yml

```diff
@@ -11,23 +11,62 @@ on:
       DOCKER_REFERENCE:
         description: 'wazuh-docker reference'
         default: 'v4.10.0'
+        required: false
+      PRODUCTS:
+        description: 'Comma-separated list of the image names to build and push'
+        default: 'wazuh-manager,wazuh-dashboard,wazuh-indexer'
+        required: true
+      FILEBEAT_MODULE_VERSION:
+        description: 'Filebeat module version'
         required: true
+      REVISION:
+        description: 'Package revision'
+        default: '1'
+        required: true
+      PUSH_IMAGES:
+        description: 'Push images'
+        type: boolean
+        default: true
+        required: true
+      ID:
+        description: "ID used to identify the workflow uniquely."
+        type: string
+        required: false
+  workflow_call:
+    inputs:
+      IMAGE_TAG:
+        description: 'Docker image tag'
+        default: '4.10.0'
+        required: true
+        type: string
+      DOCKER_REFERENCE:
+        description: 'wazuh-docker reference'
+        default: 'v4.10.0'
+        required: false
+        type: string
       PRODUCTS:
         description: 'Comma-separated list of the image names to build and push'
         default: 'wazuh-manager,wazuh-dashboard,wazuh-indexer'
         required: true
+        type: string
       FILEBEAT_MODULE_VERSION:
         description: 'Filebeat module version'
         required: true
+        type: string
       REVISION:
         description: 'Package revision'
         default: '1'
         required: true
+        type: string
       PUSH_IMAGES:
         description: 'Push images'
         type: boolean
         default: true
         required: true
+      ID:
```

Comment:
The `ID` input should be lowercase: `id`.

Reply:
Done ✅